File: MagicImages/src/main/java/com/assignment/image/MagicImages/Model/Photo.java
package com.assignment.image.MagicImages.Model;

import java.io.Serializable;

public class Photo implements Serializable {

	private long id;
	private long createdAt;
	private String name;
	private String imageUrl;

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public long getCreatedAt() {
		return createdAt;
	}

	public void setCreatedAt(long createdAt) {
		this.createdAt = createdAt;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getImageUrl() {
		return imageUrl;
	}

	public void setImageUrl(String imageUrl) {
		this.imageUrl = imageUrl;
	}

}

File: MagicWebApp/src/app/model/Photo.ts
export class Photo {
    id: number;
    createdAt: number ;
    name: string ;
    imageUrl: String ;

    constructor(id: number, createdAt: number, name: string, imageUrl: string){
      this.id = id;
      this.createdAt = createdAt;
      this.name = name;
      this.imageUrl = imageUrl;
    }
  }
  